.dev/benchmark_train.sh

```diff
@@ -1,40 +1,40 @@
 PARTITION=$1
 
 echo 'configs/hrnet/fcn_hr18s_512x512_160k_ade20k.py' &
-GPUS=4  GPUS_PER_NODE=4  CPUS_PER_TASK=2 ./tools/slurm_train.sh $PARTITION fcn_hr18s_512x512_160k_ade20k configs/hrnet/fcn_hr18s_512x512_160k_ade20k.py --options checkpoint_config.max_keep_ckpts=1 dist_params.port=24727 >/dev/null &
+GPUS=4  GPUS_PER_NODE=4  CPUS_PER_TASK=2 ./tools/slurm_train.sh $PARTITION fcn_hr18s_512x512_160k_ade20k configs/hrnet/fcn_hr18s_512x512_160k_ade20k.py --options checkpoint_config.max_keep_ckpts=1 dist_params.port=24727 --work-dir work_dirs/hrnet/fcn_hr18s_512x512_160k_ade20k >/dev/null &
```

Comment:
Maybe we only need --work-dir `work_dirs/hrnet`.
`--work-dir work_dirs/hrnet/fcn_hr18s_512x512_160k_ade20k`
--> 
`--work-dir work_dirs/hrnet/`